search/management/commands/reindex_solr.py

```diff
@@ -19,27 +19,79 @@
 #
 
 import logging
+import math
+
+from django.conf import settings
 from django.core.management.base import BaseCommand
+from utils.search.solr import Solr, SolrException
+from utils.text import remove_control_chars
+from utils.search.search_general import delete_sound_from_solr
 from sounds.models import Sound
-from utils.search.search_general import add_sounds_to_solr, delete_sound_from_solr
-from utils.search.solr import SolrException
 
 console_logger = logging.getLogger("console")
 
+def convert_to_solr_document(sound_obj):
```

Comment:
Now the function `convert_to_solr_document` is duplicated here and in `utils.search.search_general`. I think we should have only one version of this function and make sure that it works from whatever place it is called.

Reply:
First I did that, but then I moved it here because in one case I have to convert the sound object to a dict and in the other case I have to deal with the object. After that the logic if different for the 2 cases, so I think it makes more sense to have 2 methods.